core/src/main/java/org/apache/struts2/util/fs/JBossFileManager.java

```diff
@@ -210,4 +187,33 @@ private void addIfAbsent(List<URL> urls, URL fileUrl) {
         }
     }
 
+    public static class JBossFileManagerSupportUtil {
```

Comment:
@aleksandr-m 
Changed to static methods.
(don't know why i didn't just do that from the start)